test/list.spec.js

```diff
@@ -0,0 +1,37 @@
+import { mount } from '@vue/test-utils';
+import list from '../src/components/list.vue';
+import listitem from '../src/components/listitem.vue'
+
+describe('list.vue', () => {
+  const items = [{text: '1'}, {text: '2'}, {text:'3'}]
+  const wrapper = mount(list, {
+    propsData: { items: items }
+  });
+  it('renders a list', () =>{
+    expect(wrapper.contains('div')).toBe(true);
+    expect(wrapper.findAll('p')).toHaveLength(items.length)
+  });
+  const buttons = wrapper.findAll('button');
+  const addButton = buttons.at((buttons.length-1));
+
+  describe('when the add entry button is pressed', () => {
+    addButton.trigger('click');
+    it('emits an addEntry event', () => {
+      expect(wrapper.emitted('addEntry'));
```

Comment:
Yep, that's better than `wrapper.vm`. The emitted events are part of the interface of the component.